test/nodetool/test_scrub.py

```diff
@@ -58,23 +58,23 @@ def test_scrub_non_existent_keyspace(nodetool):
 # We don't test all values for --mode and --qurantine-mode, they are passed as-is
 # to the REST API, so their value make no difference when testing nodetool itself.
 @pytest.mark.parametrize("table", [[], ["tbl1"], ["tbl1", "tbl2"]])
-@pytest.mark.parametrize("mode", [None, ("-m", "ABORT"), ("--mode", "ABORT"), "ABORT"])
+@pytest.mark.parametrize("build_mode", [None, ("-m", "ABORT"), ("--mode", "ABORT"), "ABORT"])
```

Comment:
I think this rename is wrong - this is explicitly not the "build mode" - it's some sort of scrub mode. But I'm not an expert in this test.

And this rename would not be needed if you didn't give your fixture the name "mode".

Reply:
Renamed to scrub_mode.

Reply:
Why rename it at all?
You wouldn't need to rename it if you didn't create a fixture with the vague and short name "mode".
My proposal would be to rename that fixture, and don't change tests that don't need changing at all.

Reply:
Fixture is already renamed. But this is renamed for clarity.

Reply:
reverted